src/integration/Setup/SetupScript.php

```diff
@@ -122,4 +128,19 @@ public function run()
 
         echo "Installation complete\n";
     }
+
+    // Copied from https://stackoverflow.com/questions/3493253/how-to-drop-all-tables-in-database-without-dropping-the-database-itself/3493398
+    protected function dropDbTables()
+    {
+        $mysqli = new mysqli($this->host, $this->user, $this->pass, $this->name);
```

Comment:
If we're just running MySQL queries, can't we just use the Laravel Query builder with raw statements?

It seems a bit odd to create a `mysqli` object just for this. Also does this add a new system requirement?

Reply:
Can't we just copy what Laravel has in its `migrate:fresh` command?